src/controllers/update/beneficial.owner.type.controller.ts

```diff
@@ -22,9 +25,9 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
     const appData: ApplicationData = getApplicationData(req.session);
 
     if (!hasFetchedBoAndMoData(appData)) {
-      await getCompanyOfficers(req, appData[EntityNumberKey] as string);
       await getCompanyPsc(req, appData[EntityNumberKey] as string);
-      setFetchedBoMoData(appData);
+      // TO-DO: remove first call to setFetchedBoMoData in either retrieveManagingOfficers or retrieveBeneficialOwners, whatever one is called first
```

Comment:
Address TODO by setting the flags here (setFetchedBoMoData(appData)) and not in the two retrieve methods